@siddarthkay here is the error I get without this fix:

```
tools\generate-handlers\main.go: go run main.go (generated, noplugin, 3512ms)
# github.com/mutecomm/go-sqlcipher/v4
mbtls_aes.c: In function 'mbedtls_aes_setkey_enc':
mbtls_aes.c:389:8: error: implicit declaration of function 'MBEDTLS_BSWAP32'; did you mean 'MBEDTLS_BYTE_3'? [-Wimplicit-function-declaration]
  389 |      ? MBEDTLS_BSWAP32(mbedtls_get_unaligned_uint32((data) + (offset))) \
      |        ^~~~~~~~~~~~~~~
mbtls_aes.c:420:17: note: in expansion of macro 'MBEDTLS_GET_UINT32_LE'
  420 |         RK[i] = MBEDTLS_GET_UINT32_LE(key, i << 2);
      |                 ^~~~~~~~~~~~~~~~~~~~~
make[1]: *** [Makefile:351: statusgo-shared-library] Error 1
make: *** [Makefile:480: vendor/status-go/build/bin/libstatus.dll] Error 2
```

--- a/Makefile
+++ b/Makefile
@@ -78,6 +78,7 @@ ifeq ($(detected_OS),Darwin)
  GOBIN_SHARED_LIB_EXT := dylib
  LIB_EXT := dylib
  GOBIN_SHARED_LIB_CFLAGS := CGO_ENABLED=1 GOOS=darwin
+ CGO_CFLAGS+=-I/$(JAVA_HOME)/include -I/$(JAVA_HOME)/include/darwin
 else ifeq ($(detected_OS),Windows)
  GOBIN_SHARED_LIB_EXT := dll
  LIB_EXT := dll
@@ -86,8 +87,6 @@ else ifeq ($(detected_OS),Linux)
  LIB_EXT := so
  CGO_LDFLAGS += "-Wl,-soname,libstatus.so.0"
 endif
-
-CGO_CFLAGS+=-I/$(JAVA_HOME)/include -I/$(JAVA_HOME)/include/darwin
 export GOPATH ?= $(HOME)/go
 
 GIT_ROOT ?= $(dir $(realpath $(lastword $(MAKEFILE_LIST))))
@@ -139,6 +138,12 @@ endif
 LIBSDS := $(NIM_SDS_LIB_DIR)/libsds.$(LIB_EXT)
 CGO_CFLAGS+=-I$(NIM_SDS_INC_DIR)
 CGO_LDFLAGS+=-L$(NIM_SDS_LIB_DIR) -lsds
+
+# mbedtls configuration for go-sqlcipher
+ifeq ($(detected_OS),Windows)
+ # On Windows, use portable C implementations and add -Werror=implicit-function-declaration workaround
+ CGO_CFLAGS+=-Wno-implicit-function-declaration
+endif
 
 # Common flags
 

